packages/module/src/ErrorBoundary/ErrorStack.tsx

```diff
@@ -1,16 +1,32 @@
 import * as React from 'react';
 import { Text } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+import { global_BackgroundColor_dark_300 as backgroundDark } from '@patternfly/react-tokens/dist/js/global_BackgroundColor_dark_300';
+import { global_icon_FontSize_md as iconMd } from '@patternfly/react-tokens/dist/js/global_icon_FontSize_md';
+import { global_spacer_sm as spacerSm } from '@patternfly/react-tokens/dist/js/global_spacer_sm';
 
 interface ErrorStackProps {
   error: Error;
 }
 
-const errorStackClass = 'ins-error-boundary-stack';
+const useStyles = createUseStyles({
+  errorStack: {
+    fontFamily: 'monospace',
+    fontSize: iconMd.value,
+    textAlign: 'left',
+    backgroundColor: 'white',
+    borderStyle: 'solid',
+    borderColor: backgroundDark.value,
```

Comment:
I think you can just use the `'var(--pf-...)'` strings instead of these tokens. It might be safer than importing the values.

Reply:
@Hyperkid123 thank you, pushed the update